dev/src/Command/AddComponentCommand.php

```diff
@@ -111,16 +132,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ->render();
         }
 
-        $productHomepage = $this->getHelper('question')->ask(
+        $productDocumentation = null;
+        $yamlFileContent = $this->loadYamlConfigContent($new, dirname($proto));
+        if (!empty($yamlFileContent)) {
+            $productDocumentation = $yamlFileContent['publishing']['documentation_uri'] ?? null;
+        }
```

Comment:
nit, but since you're already checking the existence of `$yamlFileContent`, the `if (is_null(...))` check is unnecessary.
```suggestion
        $yamlFileContent = $this->loadYamlConfigContent($new, dirname($proto));
        $productDocumentation = $yamlFileContent['publishing']['documentation_uri'] ?? null;
```